app/helpers/administrate/application_helper.rb

```diff
@@ -4,5 +4,17 @@ def render_field(field, locals = {})
       locals.merge!(field: field)
       render locals: locals, partial: field.to_partial_path
     end
+
+    def display_resource_name(resource_name)
+      resource_name.
+        to_s.
+        classify.
+        constantize.
+        model_name.
+        human(
+          count: 0,
```

Comment:
@joelq this dictates how the resource names show up in the sidebar, and in the header on the index page. I think in both of those cases, we always want the displayed name to be in the plural form.

![customers___administrate_prototype](https://cloud.githubusercontent.com/assets/829576/11634891/380c7f7c-9ce1-11e5-98c1-e34a6b877b14.png)

I started to code out an argument to this function that would let you specify a custom count, but I think YAGNI applies to that. If we need to display the singular form in the future, we can adjust this method accordingly.